.gitpod.yml

```
@@ -2,6 +2,24 @@
 tasks:
   - init: make bootstrap
     command: make watch
+  - openMode: split-right
+    command: |-
+      clear && echo '
+      Welcome to Gitpod!
+
+      If you haven'\''t forked Babel yet and you want to prepare a PR, you can use
+      the Ctrl+P (or Cmd+P) shortcut, type `>Fork` and press ENTER: Gitpod\ will
```

```suggestion
      the Ctrl+P (or ⌘P) shortcut, type `>Fork` and press ENTER: Gitpod\ will
```
Introducing a bit Mac taste.

---

Thanks!